# Goal of this action is to perform the path
vecfield_control/Path path  # Specify the path to perform
---
# Define the result
bool goal_reached
---
# Define a feedback message
float32 percent_complete
